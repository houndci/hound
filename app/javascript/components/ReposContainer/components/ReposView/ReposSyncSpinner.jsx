import React from 'react';

export default class ReposSyncSpinner extends React.Component {
  render() {
    return (
      <div className="repos-syncing">
        <div className="dot"></div>
        <div className="dot"></div>
        <div className="dot"></div>
      </div>
    );
  }
}
