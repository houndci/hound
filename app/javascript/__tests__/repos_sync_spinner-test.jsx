import ReposSyncSpinner from '../components/ReposContainer/components/ReposView/ReposSyncSpinner';

it('renders appropriately', () => {
  const wrapper = shallow(
    <ReposSyncSpinner />
  );
  expect(wrapper).toMatchSnapshot();
});
