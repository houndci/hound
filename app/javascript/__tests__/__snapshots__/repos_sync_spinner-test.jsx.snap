// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders appropriately 1`] = `
<div
  className="repos-syncing"
>
  <div
    className="dot"
  />
  <div
    className="dot"
  />
  <div
    className="dot"
  />
</div>
`;
